imconfig.h

```diff
@@ -13,19 +13,16 @@
 
 #pragma once
 
-extern "C" void iggAssert(int result, char const *expression, char const *file, int line);
```

Comment:
removing `iggAssert` and the macro below removes functionality to capture assertions within ImGui in Go.
This should not be removed.